### PR TITLE
PE-697 - Pin packages version to the project requirements.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR pins the current version of the packages to the project requirements.
Adds a safe check to a dict, to avoid KeyError exception.

## Reviewers: 

- [ ] @andrey-canon

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -28,24 +28,23 @@ def findfiles(pat):
             ],
         },
     install_requires=[
-        'path.py',
-        'argparse',
```
Comment (andrey-canon) [inline setup.py:32]: why did you remove this ?

File: setup.py [hunk 2]
```diff
@@ -28,24 +28,23 @@ def findfiles(pat):
...
+        'pygeoip==0.3.2',
+        'pytz==2019.1',
+        'python-dateutil==2.8.0',
+        'geoip2==2.9.0',
+        'lxml==4.3.4',
+        'BeautifulSoup==3.2.1',
+        'unicodecsv==0.14.1',
+        'Jinja2==2.10.1',
+        'google-api-python-client==1.7.9',
         'edxcut',
```
Comment (andrey-canon) [inline setup.py:41]: why was this excluded ?

File: setup.py (hunk 1 above)
Comment (Squirrel18) [reply setup.py:32]: argparse is part of the Standard library. @andrey-canon

File: setup.py (hunk 2 above)
Comment (Squirrel18) [reply setup.py:41]: edX2bigquery requires installing edxcut by hand, since edxcut has a problem with its setup package. @andrey-canon

**Review by andrey-canon — APPROVED** [review]
